Kne baseline config

cprabha opened the pull request:
Hi,

I have added baseline config for kne devices config file.
Please could merge it.

cprabha committed:
added baseline config

cprabha committed:
Merge branch 'main' into config_ncptx

cprabha committed:
Merge branch 'main' into config_ncptx

cprabha committed:
Merge branch 'main' into config_ncptx

cprabha committed:
Merge branch 'main' into config_ncptx

navaneethyv reviewed (changes requested):
This is a valid comment, if you have taken care of this please make a note of you having taken care of this, and close, if not can you please add a case as to why this is needed and how you arrive at this

cprabha commented on topologies/kne/juniper/ncptx/config.cfg:
```diff
@@ -248,6 +248,11 @@ routing-options {
         oc-tlv-support;
         export balance;
     }
+    programmable-rpd {
+        gribi {
+            max-modify-resp-delay 105;
+        }
+    }
```
This is config on router and not the code change. 
This particular config is part of baseline configuration on physical setup, we are adding here for kne topology,

cprabha commented:
> I have added baseline config for kne devices config file.

@navaneethyv Changes which i have added in this gitpull is already part of baseline configuration and this is not code change.  hence did not add any comment in cli , t have made a note in pull request description .

navaneethyv committed:
Merge branch 'main' into config_ncptx

cprabha committed:
Update config.cfg

updating baseconfig for junuper

cprabha reviewed (commented):
updated baseconfig for kne

cprabha committed:
Merge branch 'main' into config_ncptx
